### PR TITLE
Feature/3418 drawer footer playground

### PR DESCRIPTION
**Pull request by ektasawant**
Fixes 3418 .

#### Changes proposed in this Pull Request:
- Footer Playground
- Footer hideContentOnCollapse prop issue fixed [here](https://github.com/brightlayer-ui/react-component-library/pull/495)
- **Need to merge after [Drawer Subheader playground PR]**(https://github.com/brightlayer-ui/react-component-library/pull/494)

#### Screenshots:
Screenshot 2022-09-02 at 3 23 18 PM

#### To Test:
- http://localhost:3001/components/drawer-footer/playground

### CODE REVIEW

**Comment by emclaug2** [comment]
I'm re-running the deploy action now.

**Comment by daileytj** [comment]
Screen Shot 2022-09-06 at 11 15 16 AM

Code snippet should be pinned to the bottom regardless of screen height.

**Comment by daileytj** [comment]
https://user-images.githubusercontent.com/13989985/188672741-c7a8f2c1-c18f-4ab4-8559-039e5976a6d2.mov

Quite a bit of movement when toggling props and the `hideContentOnCollapse` isn't working for the footer as expected

File: docs/src/componentDocs/DrawerFooter/playground/DrawerFooterTypes.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import { ComponentType } from '../../../__types__';
+import * as Colors from '@brightlayer-ui/colors';
+
+export const drawerTypes: ComponentType = {
```
Comment (daileytj) [inline docs/src/componentDocs/DrawerFooter/playground/DrawerFooterTypes.tsx:4]: I'm still a bit confused by all of these "Types". Did we decide to rename these to "Config" instead? e.g. drawerTypes => drawerConfig

Same goes for the file name.

If this still needs addressed across all components we can address it in a separate story.

Comment (ektasawant) [reply docs/src/componentDocs/DrawerFooter/playground/DrawerFooterTypes.tsx:4]: > Types

We didn't but I can make changes in next PR.

**Comment by ektasawant** [comment]
> Screen.Recording.2022-09-06.at.11.16.14.AM.mov 
> Quite a bit of movement when toggling props and the `hideContentOnCollapse` isn't working for the footer as expected
This is fixed here https://github.com/brightlayer-ui/react-component-library/pull/495

**Comment by ektasawant** [comment]
> Screen Shot 2022-09-06 at 11 15 16 AM
> 
> Code snippet should be pinned to the bottom regardless of screen height.

Can you clear cache and load this URL?

**Review by daileytj — APPROVED** [review]

**Pull request merged**